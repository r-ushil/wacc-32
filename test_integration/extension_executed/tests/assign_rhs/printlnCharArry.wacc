# You cannot directly print a char[] in WACC

# Output:
# Hello World!

# Exit:
# 0

# Program:

begin
  println ['H','e','l','l','o',' ','W','o','r','l','d','!'];

  exit 5
end

# EXTENSION HAS MADE THIS ONCE INVALID TEST CASE VALID
