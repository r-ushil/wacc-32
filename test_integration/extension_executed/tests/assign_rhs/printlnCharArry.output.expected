Compiler exit code: 0
--------- stdout ---------
Hello world!
--------------------------
Exit code: 0
